tests/sdl2_net_client.c

```diff
@@ -46,11 +46,9 @@ void finish(int result) {
     SDLNet_Quit();
   }
 #ifdef __EMSCRIPTEN__
-  REPORT_RESULT(result);
-  emscripten_force_exit(result);
-#else
-  exit(result);
+  emscripten_cancel_main_loop();
 #endif
+  exit(result);
 }
```

Comment:
unrelated change? from here the patch seems to be general cleanups and not C++ namespaces as the title says

Reply:
Ooops, sorry about that.  rebased